code/data_processor_block/data_processor.py

```diff
@@ -0,0 +1,60 @@
+#Jakub Podolak for Plastic Monkeys Cansat 2019
+
+import graphyte
+from time import sleep
+import ast
+import configparser
+import logging
+import sys
+
+config_file = sys.argv[1]
+config = configparser.ConfigParser()
+config.read(config_file)
+
+graphyte_address = config['SETTINGS']['graphyte_address']
+graphyte_prefix = config['SETTINGS']['graphyte_prefix']
+sleep_time = float(config['SETTINGS']['sleep_time'])
+
+graphyte.init(graphyte_address, prefix=graphyte_prefix)
+
+#open log file
+logging.basicConfig(filename="data_processor.log",level=logging.INFO)
+
+#function to read paths to input logs from config file
+def get_directories(config):
+    paths = config.items("PATHS")
+    directories = [second for first,second in paths]
```

Comment:
this is just cosmetics, but we don't need to assign this list to variable if we're not gonna use that anywhere else, that means we could just return the list comprehension like this:
```python
def get_directories(config):
	    paths = config.items("PATHS")
	    return [second for first, second in paths]
```

Reply:
Yes, of course. I have some kind of instinct to make everything legible, sometimes in the wrong way making things actually too complicated.

Reply:
Resolved here: ed4701c